### PR TITLE
Add zh-{TW,CN} translation for 2fa setting

### PR DESCRIPTION
**Pull request by johnlinvc**

### CODE REVIEW

File: config/locales/zh-CN.yml [hunk 1]
```diff
@@ -154,32 +154,32 @@ zh-CN:
...
-      key:
-      time_based:
+      title: 启用多重要素验证
+      scan_prompt: 请用你的验证装置扫描 QR-code。如果你没办法扫描，手动输入下面的资料。
+      otp_prompt: 输入验证装置上的数字来继续。
+      confirm: 我已把复原码收在安全的地方。
+      enable: 启用
+      account: "帐号: %{account}"
+      key: "金钥: %{key}"
+      time_based: "Time based: 是"
```
Comment (ecnelises) [inline config/locales/zh-CN.yml]: Time-based => 基于时间的
Google authenticator uses such translation.

File: config/locales/zh-CN.yml [hunk 2]
```diff
@@ -203,16 +203,16 @@ zh-CN:
...
+        update: 更新
         level:
-          title:
-          disabled:
-          ui_only:
-          ui_and_api:
+          title: 多重验证等级
+          disabled: 停用
+          ui_only: 只有使用者介面
+          ui_and_api: 使用者介面與API
```
Comment (ecnelises) [inline config/locales/zh-CN.yml]: 與 => 与

File: config/locales/zh-CN.yml [hunk 3]
```diff
@@ -154,32 +154,32 @@ zh-CN:
...
+      otp_prompt: 输入验证装置上的数字来继续。
+      confirm: 我已把复原码收在安全的地方。
+      enable: 启用
+      account: "帐号: %{account}"
+      key: "金钥: %{key}"
+      time_based: "Time based: 是"
     create:
-      qrcode_expired:
-      success:
+      qrcode_expired: QR-code 和金钥已过期。请再重新注册装置。
```
[the remark this answers is not in the document]
Comment (ecnelises) [reply config/locales/zh-CN.yml]: 金钥 => 密钥

File: config/locales/zh-CN.yml [hunk 4]
```diff
@@ -203,16 +203,16 @@ zh-CN:
         delete_profile: 删除个人资料
         warning: 警告
       mfa:
-        multifactor_auth:
-        disabled:
-        go_settings:
-        enabled:
-        update:
+        multifactor_auth: 多重要素验证
```
Comment (ecnelises) [inline config/locales/zh-CN.yml]: Would `多因素验证` be better?

File: config/locales/zh-CN.yml [hunk 5]
```diff
@@ -13,8 +13,8 @@ zh-CN:
   none: 无
   not_found: 未找到
   please_sign_up: 拒绝访问。请先在 https://rubygems.org 上注册一个账号。
-  otp_incorrect:
-  otp_missing:
+  otp_incorrect: 你的 OTP 码 不正确。请检查后再试一次。
+  otp_missing: 你已启用多重要素验证，但是没有输入 OTP 码。请输入后再试一次。
```
[the remark this answers is not in the document]
Comment (johnlinvc) [reply config/locales/zh-CN.yml]: MSDN suggests 多重身份验证
https://www.microsoft.com/en-us/language/Search?&searchTerm=Multifactor&langID=124&Source=true&productid=0

**Comment by johnlinvc** [comment]
Fixed the suggestions. But still not sure which official name we should use.
Also rebased to latest master.

File: config/locales/zh-CN.yml (hunk 4 above)
Comment (sonalkr132) [reply config/locales/zh-CN.yml]: this Wikipedia article may be helpful https://zh.wikipedia.org/wiki/%E5%A4%9A%E9%87%8D%E8%A6%81%E7%B4%A0%E9%A9%97%E8%AD%89

**Comment by ecnelises** [comment]
My subjective view:
`多重身份认证` or `多重认证` sounds more smooth.
`多重要素认证` fits English meaning better.

I'll have a look at what most sites use. Thanks for your contribution! @johnlinvc

File: config/locales/zh-TW.yml [hunk 6]
```diff
@@ -175,32 +175,32 @@ zh-TW:
...
-      success:
+      qrcode_expired: QR-code 和金鑰已過期。請再重新註冊裝置。
+      success: 你已成功啟用多重要素驗證。
     recovery:
-      continue:
-      title:
-      note:
+      continue: 繼續
+      title: 復原碼
+      note: 你必須要把復原碼收在安全的地方來避免失去帳號。失去認證裝置的情況下，每一個復原碼可以使用一次。
```
Comment (khiav223577) [inline config/locales/zh-TW.yml]: I would say "請務必將復原碼收在安全的地方以避免失去帳號"
```suggestion
      note: 請務必將復原碼收在安全的地方以避免失去帳號。失去認證裝置的情況下，每一個復原碼可以使用一次。
```

File: config/locales/zh-TW.yml [hunk 7]
```diff
@@ -175,32 +175,32 @@ zh-TW:
...
-      key:
-      time_based:
+      title: 啟用多重要素驗證
+      scan_prompt: 請用你的驗證裝置掃描 QR-code。如果你沒辦法掃描，手動輸入下面的資料。
+      otp_prompt: 輸入驗證裝置上的數字來繼續。
+      confirm: 我已把復原碼收在安全的地方。
+      enable: 啟用
+      account: "帳號: %{account}"
+      key: "金鑰: %{key}"
+      time_based: "Time based: 是"
```
Comment (khiav223577) [inline config/locales/zh-TW.yml]: How about this?
```suggestion
      time_based: "基於時間的: 是"
```

**Pull request merged**